en/admin/console_commands.md

```diff
@@ -34,6 +34,7 @@ Custom to wallabag:
  - `wallabag:import:redis-worker`: Useful if you use Redis.
  - `wallabag:install`: (re)Install wallabag
  - `wallabag:tag:all`: Tag all entries for an user using his/her tagging rules.
+ - `wallabag:user:show`: Shows the details for an user.
```

Comment:
Is that an exception? A `a` before a word starting with a vowel shouldn't be converted to `an`?